src/components/Session/Deposit/DepositCurrency.jsx

```diff
@@ -18,6 +33,8 @@ export default class DepositCurrency extends React.Component {
           options={options}
           value={selectedValue}
           onChange={(newValue) => this.props.onCurrencyChange(newValue.value)}
+          optionRenderer={this._optionRenderer.bind(this)}
```

Comment:
Instead of using bind and having this._optionRenderer, can you just make it a big arrow function inside render() {.

Although slightly less efficient, theres less cognitive overload

Reply:
ok, that's fine